File: frontend/src/utils/units.ts
/** 1 Hartree (atomic unit of energy) in electron-volts. */
export const HARTREE_TO_EV = 27.2114

/** Convert an energy in atomic units to eV, formatted to 2 decimal places. */
export function auToEv(au: number): string {
  return (au * HARTREE_TO_EV).toFixed(2)
}

/** 1 Bohr radius (atomic unit of length) in Ångströms. */
export const BOHR_TO_ANGSTROM = 0.529177

/** Convert a length in atomic units to Å, formatted to 2 decimal places. */
export function auToAngstrom(au: number): string {
  return (au * BOHR_TO_ANGSTROM).toFixed(2)
}

File: backend/shared/units.py
"""Atomic unit constants used throughout the solver.

In atomic units: ħ = m_e = e = 1.
All energies in Hartree, lengths in Bohr radii (a₀), times in atomic units
(1 a.u. ≈ 24.19 attoseconds).
"""

HBAR: float = 1.0       # reduced Planck constant
M_E: float = 1.0        # electron mass
E_CHARGE: float = 1.0   # elementary charge

A0_TO_ANGSTROM: float = 0.529177210903   # Bohr radius in Ångström
HARTREE_TO_EV: float = 27.211386245988   # Hartree in electron-volts
AU_TIME_TO_AS: float = 24.18884326505    # atomic time unit in attoseconds
